ceph_deploy/tests/test_keys_equivalent.py

```diff
@@ -0,0 +1,171 @@
+        key_path_02 = self.test_dir + "/02.keyring"
+        write_key_mon_without_caps(key_path_01, secret_01)
+        write_key_mon_without_caps(key_path_02, secret_01)
+        same = gatherkeys._keyring_equivalent(key_path_01, key_path_02)
+        assert same == True
+
+
+    def test_different_without_caps(self):
+        secret_01 = new.generate_auth_key()
+        secret_02 = new.generate_auth_key()
+        key_path_01 = self.test_dir + "/01.keyring"
+        key_path_02 = self.test_dir + "/02.keyring"
+        write_key_mon_without_caps(key_path_01, secret_01)
+        write_key_mon_without_caps(key_path_02, secret_02)
+        same = gatherkeys._keyring_equivalent(key_path_01, key_path_02)
+        assert same == False
+
+
+    def test_identical_mixed_caps(self):
+        secret_01 = new.generate_auth_key()
+        key_path_01 = self.test_dir + "/01.keyring"
+        key_path_02 = self.test_dir + "/02.keyring"
+        write_key_mon_with_caps(key_path_01, secret_01)
+        write_key_mon_without_caps(key_path_02, secret_01)
+        same = gatherkeys._keyring_equivalent(key_path_01, key_path_02)
+        assert same == True
+
+
+    def test_different_mixed_caps(self):
+        secret_01 = new.generate_auth_key()
+        secret_02 = new.generate_auth_key()
+        key_path_01 = self.test_dir + "/01.keyring"
+        key_path_02 = self.test_dir + "/02.keyring"
+        write_key_mon_with_caps(key_path_01, secret_01)
+        write_key_mon_without_caps(key_path_02, secret_02)
+        same = gatherkeys._keyring_equivalent(key_path_01, key_path_02)
+        assert same == False
+
+
+    def test_identical_caps_mixed_tabs(self):
+        secret_01 = new.generate_auth_key()
+        key_path_01 = self.test_dir + "/01.keyring"
+        key_path_02 = self.test_dir + "/02.keyring"
+        write_key_mon_with_caps(key_path_01, secret_01)
+        write_key_mon_with_caps_with_tab(key_path_02, secret_01)
+        same = gatherkeys._keyring_equivalent(key_path_01, key_path_02)
+        assert same == True
+
+
+    def test_different_caps_mixed_tabs(self):
+        secret_01 = new.generate_auth_key()
+        secret_02 = new.generate_auth_key()
+        key_path_01 = self.test_dir + "/01.keyring"
+        key_path_02 = self.test_dir + "/02.keyring"
+        write_key_mon_with_caps(key_path_01, secret_01)
+        write_key_mon_with_caps_with_tab(key_path_02, secret_02)
+        same = gatherkeys._keyring_equivalent(key_path_01, key_path_02)
+        assert same == False
+
+
+    def test_identical_caps_mixed_quote(self):
+        secret_01 = new.generate_auth_key()
+        key_path_01 = self.test_dir + "/01.keyring"
+        key_path_02 = self.test_dir + "/02.keyring"
+        write_key_mon_with_caps_with_tab(key_path_01, secret_01)
+        write_key_mon_with_caps_with_tab_quote(key_path_02, secret_01)
+        same = gatherkeys._keyring_equivalent(key_path_01, key_path_02)
+        assert same == True
```

Comment:
boolean assertions should be written as:

    same is True

Or:

    same is False